javascript/javascript3/week2/movies.js

```diff
@@ -0,0 +1,13 @@
+async function moviesExerices() {
+  const getResponse = await fetch(
```

Comment:
It's good practice to handle errors when `await` fails. For example see the code example bellow

```
async function some_work(){

   try{
// Let's create a fake problem      
      await abc;
   }

   catch(error){
      console.log("abc does not exist", error)
   }

   finally{
      console.log("Runs code anyways")
   }
}

some_work()  // run the code
```